test/mail_helper.js

```diff
@@ -56,6 +57,9 @@ module.exports = (printLogs) => {
             if (vc) {
               console.log('\x1B[32m', link, '\x1B[39m')
             }
+            else if (sc) {
+              console.log('\x1B[32mSign-in code: ', sc, '\x1B[39m')
```

Comment:
As above, this may be a little confusing because "signin code" is already the name of something else.  Could we call this e.g. "verification code" or similar?